packages/langium-cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "langium-cli",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Please don't increase the version number now. We'll do it during the publishing process.

Reply:
@spoenemann 
I reverted this change